tools/wptrunner/wptrunner/testloader.py

```diff
@@ -467,7 +467,8 @@ def __init__(self,
                  total_chunks=1,
                  chunk_number=1,
                  include_https=True,
-                 skip_timeout=False):
+                 skip_timeout=False,
+                 executor_classes={}):
 
         self.test_types = test_types
```

Comment:
This class should have docs

Reply:
Agreed. I've filed an issue so that can be addressed independently: https://github.com/web-platform-tests/wpt/issues/12384